bench/wtperf/runners/multi-btree-backup.wtperf

```diff
@@ -0,0 +1,18 @@
+# wtperf options file: small btree multi-database configuration
+# Original cache was 500MB.  Shared cache is 500MB * database_count.
+conn_config="shared_cache=(name=pool,size=2500MB,chunk=1M),log=(enabled=false)"
+#conn_config="cache_size=250MB,log=(enabled=false)"
```

Comment:
Should we remove that line ?

Reply:
I think we should be good with keeping this.

Reply:
The commented line is not required. Please remove that line from the original `multi-btree.wtperf` config also.